docs/FHIRServerUsersGuide.md

```diff
@@ -85,7 +85,7 @@ Watson Health FHIR Server version 2.1 was developed under the Watson Health.
 
 ## 2.1 Installing a new server
 1.	To install the FHIR server, first obtain the installation package `fhir-install-<version>.zip`.
-You can download the package from the FHIR server team's [Artifactory server](https://na.artifactory.swg-devops.com/artifactory/webapp/#/artifacts/browse/simple/General/wh-fhir-server-releases-maven-local/com/ibm/watsonhealth/fhir/fhir-install).
+You can download the package from the FHIR server team's [Artifactory server](https://na.artifactory.swg-devops.com/artifactory/webapp/#/artifacts/browse/simple/General/wh-fhir-server-releases-maven-local/com/ibm/watson/health/fhir/fhir-install).
```

Comment:
this whole section needs to be removed as it points to internal artifacts inside IBM

Reply:
right - do you want me to remove?

Reply:
yes please

Reply:
```suggestion
The Maven build creates the zip package under `fhir-install/target`. Alternatively, releases will be available from the [Releases tab](https://github.com/ibm/fhir/releases).
```

Reply:
also, the prevous line should say `fhir-server-distribution.zip` and not `fhir-install-<version>.zip`